src/main/java/org/codeforamerica/messaging/models/MessageBatch.java

```diff
@@ -26,13 +25,12 @@ public class MessageBatch {
     Template template;
     @JsonIgnore
     private byte[] recipients;
-    @OneToMany(mappedBy = "messageBatch", fetch = FetchType.LAZY, cascade = CascadeType.ALL)
-    List<Message> messages;
     @Future
     OffsetDateTime sendAt;
     @CreationTimestamp
     private OffsetDateTime creationTimestamp;
     @UpdateTimestamp
     private OffsetDateTime updateTimestamp;
-
+    @Transient
+    MessageBatchMetrics metrics;
```

Comment:
What does this look like in json? Does it include the labels or just the integer counts?

Reply:
It looks like this:
```
{
    "creationTimestamp": "2023-06-09T11:53:18.383119-07:00",
    "id": 10,
    "metrics": {
        "acceptedEmailCount": 2,
        "acceptedSmsCount": 2,
        "deliveredEmailCount": 0,
        "deliveredSmsCount": 0,
        "rejectedEmailCount": 0,
        "rejectedSmsCount": 0,
        "undeliveredEmailCount": 0,
        "undeliveredSmsCount": 0
    },
    "sendAt": null,
    "template": {
        "creationTimestamp": "2023-05-22T15:20:33.578697-07:00",
        "id": 1,
        "name": "hi",
        "templateVariants": [
            {
                "body": "Hello {{ color }} there",
                "id": 2,
                "language": "en",
                "subject": "Hi",
                "treatment": "A"
            },
            {
                "body": "Cómo tú estás",
                "id": 1,
                "language": "es",
                "subject": "Hi",
                "treatment": "A"
            }
        ],
        "updateTimestamp": null
    },
    "updateTimestamp": "2023-06-09T11:53:18.383164-07:00"
}
```